src/core/Visualizer/index.tsx

```diff
@@ -1,11 +1,18 @@
 import { CSSProperties, type ReactNode } from "react";
 
+// TODO(@keiya01): Change directory structure
+import DropHolder from "@reearth/components/atoms/DropHolder";
+import Filled from "@reearth/components/atoms/Filled";
+
 import Crust, {
   type Alignment,
   type Location,
   type WidgetAlignSystem,
   type WidgetLayoutConstraint,
 } from "../Crust";
+import { ExternalPluginProps } from "../Crust/Plugins/Plugin";
+import { InternalWidget } from "../Crust/Widgets";
+import { Tag } from "../mantle/compat/types";
```

Comment:
It is good to simplify import statements by re-exporting types:

```ts
// 🤔 
import { ExternalPluginProps } from "../Crust/Plugins/Plugin";
import { InternalWidget } from "../Crust/Widgets";
import { Tag } from "../mantle/compat/types";

// 👍 
import type { ExternalPluginProps, InternalWidget } from "../Crust";
import type { Tag } from "../Map";
```

Allowing access to the internals of a component to import types at will would result in a loss of encapsulation of the component and leakage of internal implementation details to the outside world. This makes future implementation changes more and more difficult. To limit the scope of changes, it is better to make all imports and exports originate from the component's index.tsx and forbid components outside of it from directly importing the component's inner types.

Also use `type` keyward as possible as you can

Reply:
I fixed! [refactor: export/import](https://github.com/reearth/reearth-web/pull/399/commits/634009cb35dc90f241b8a152a335ba820226caca)